src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -68,6 +69,10 @@ public Api getApi() {
     return this.getService().getApis(0);
   }
 
+  public String getApiRevision() {
+    return service.getDocumentation().getOverview();
+  }
+
```

Comment:
It now is again (in CommonDiscoveryProvider): it was a somewhat arbitrary choice to use the Service documentation overview field to hold the API revision (for lack of a better option), so this reference seemed unclear otherwise.